client/components/mma/accountoverview/AccountOverview.tsx

```diff
@@ -108,15 +109,17 @@ const AccountOverviewPage = ({ isFromApp }: IsFromAppProps) => {
 	);
 
 	const productCategories = [
-		...allActiveProductDetails,
-		...allCancelledProductDetails,
-	]
-		.map(
-			(product: ProductDetail | CancelledProductDetail) =>
-				product.mmaCategory,
-		)
-		.filter((value, index, self) => self.indexOf(value) === index);
-
+		...new Set(
```

Comment:
I'm assuming the `Set` is being used here as an alternative to the previous approach of returning unique values? I wonder if that intent could be made clearer by assigning to consts with names? E.g.

```js
const allProductCategories = ...;
const uniqueProductCategories = [...new Set(allProductCategories)];
```

(I do like the `Set` approach more than the previous use of filter!)

Reply:
yeah thats a nice idea, would make it clearer whats going on 👍